### PR TITLE
flowtools + flow datatype

### PR DESCRIPTION
**Pull request by cristhomas**
* [ x] I have read the guidelines above.
* [x ] This PR adds a new recipe.

### CODE REVIEW

**Comment by cristhomas** [comment]
@bioconda/core not sure how to fix the unhappy Travis build.

**Comment by daler** [comment]
@cristhomas looks like the error is here: https://travis-ci.org/bioconda/bioconda-recipes/jobs/207100532#L715.

Probably due to a hard-coded Rscript path in checkFCS.R; try changing to `#!/usr/bin/env Rscript`

File: recipes/ig-flowtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - bioconductor-flowai
+    - bioconductor-flowdensity
+    - flock
+
+test:
+  commands:
+    - auto_collapse_pops.py -h
+
+about:
+  home: https://github.com/ImmPortDB/ig-flowtools
```
Comment (bgruening) [inline recipes/ig-flowtools/meta.yaml]: Can you also add a license?

File: recipes/ig-checkfcs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  run:
+    - 'r >=2.10.0'
+    - bioconductor-flowcore
+
+test:
+  commands:
+    - checkFCS.R
+
+about:
+  home: https://github.com/ImmPortDB/ig-checkfcs
```
Comment (bgruening) [inline recipes/ig-checkfcs/meta.yaml]: Can you add a license here as well?

**Comment by cristhomas** [comment]
The problem with Travis not finding bioconductor-flowai and bioconductor-flowcl I don't know how to fix, because those 2 packages are in bioconda already.

**Comment by acaprez** [comment]
@cristhomas - the OS X build failed because those two package are currently there only for linux.  I have a couple of PR's in (#4007, #4008) to add them.

**Comment by cristhomas** [comment]
@acaprez  thank you!

**Comment by cristhomas** [comment]
Is there a way to re-build with Travis without committing/closing and re-opening the PR?

**Comment by acaprez** [comment]
Yes, you can manually restart the build.  On the Travis CI page for the job (https://travis-ci.org/bioconda/bioconda-recipes/jobs/207491070), there's a "Restart job" button on the right-hand side of the page.

**Comment by acaprez** [comment]
Hmm, looking at the error seems to suggest a problem with the Miniconda download.  A build of mine just died with an HTTP 503 error a few minutes ago.  I suspect there may be networking issues somewhere between Travis and Continuum at the moment.

**Comment by cristhomas** [comment]
I don't have a 'restart job' button (or my brain is fried a lot more than I thought, which is also possible).
I'll prob wait until Monday anyway to look at this again. Thanks for your help =)

**Comment by acaprez** [comment]
Oh - you'll need to sign in first with your GitHub credentials (upper right corner).  Then Travis knows you're associated with this project and should show more options.

**Comment by bgruening** [comment]
Since yesterday the download are very unreliable :(
I will clicking the reload button until this goes through ;)

**Comment by bgruening** [comment]
Supi, Dupi! Thanks @cristhomas and @acaprez!

**Comment by cristhomas** [comment]
awesome, thank you @bgruening !

**Pull request merged**